signature/src/balrog.rs

```diff
@@ -361,14 +316,30 @@ impl<'a> Balrog<'_> {
             &pubkey, &algorithm, &sig_asn1, &message,
         )?)
     }
+
+    /* Wrapper function to verify everything at once. */
```

Comment:
I will note that you are now verifying a _little_ less than before here, because `balrog.cpp` is validating the hash of the root cert -- but I also think that it's OK to separate out verification of the signature, and verification of what made the signature.

(Mostly just calling this out in case you wanted to update comments.)

Reply:
Good callout, thank you.